api/views/workflow.py

```diff
@@ -338,7 +338,7 @@ def post(self, request, *args, **kwargs):
         data = {}
         data['source'] = request.user.id
```

Comment:
Could you remove this line? It's handled by passing request in (or at least should be)